### PR TITLE
SFB-72: start of custom codelist

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## ID
 [SFB-72](https://binnenland.atlassian.net/browse/SFB-72)

 ## Description

The user wants to have a new interface where they want to see and edit code lists. The generic code lists may not be edited but the others can. I created a new page for showing the code lists and viewing them.

This is behind a feature flag so no one will notice this when it is off.

The next step in another PR is to start creating new forms and editing them.

 ## Type of change

 - [ ] Bug fix
 - [x] New feature
 - [ ] Breaking change
 - [ ] Other

 ## How to test

1. Set the feature flag to false
2. Go to codelijsten in the ui
3. View all lists
4. See details of a list and go back

 ## Links to other PR's

 - /

### CODE REVIEW

**Comment by JonasVanHoof** [comment]
@aatauil  Im not sure whats going wrong here. We have always had a tricky pagination when running the application locally but now when going to the next page it isn't even loading the other items. Also the sorting is doing strange.. the network tab shows a call for a symbolset from appuniversum instead of a new get call with a different page..
image

Can you have a look?

File: app/templates/codelijsten/index.hbs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                  @route="codelijsten.edit"
+                  @model={{conceptScheme.id}}
+                  @ariaHidden={{true}}
+                >{{conceptScheme.label}}</AuLink>
+              </td>
+              <td class="au-u-visible-medium-up">{{conceptScheme.id}}</td>
+            </tr>
+          {{/each}}
+        </:body>
+      </AuTable>
```
[the remark this answers is not in the document]
Comment (ZijneDwaasheid) [reply app/templates/codelijsten/index.hbs:130]: also not scroll-able :)

**Review by ZijneDwaasheid — CHANGES_REQUESTED** [review]

**Comment by JonasVanHoof** [comment]
@ZijneDwaasheid  Hm alright I see where this is going wrong haha. So I have this branch and then an extended one from this. The things you are saying are fixed in https://github.com/lblod/frontend-form-builder/pull/85. 

Sorry this makes it actually a little more difficult than I expected..  Maybe you should check https://github.com/lblod/frontend-form-builder/pull/85 as final product

**Comment by JonasVanHoof** [comment]
Closing this branch as it is no more the base for the feature. The PR created of this branch is https://github.com/lblod/frontend-form-builder/pull/85 from now you can find latest here. Sorry for the big PR 🖖🏻

**Pull request closed without merging**